kotlin/feature-layer-extrusion/README.md

```diff
@@ -1,19 +1,36 @@
-# Feature Layer Extrusion
+# Feature layer extrusion
 
-This sample demonstrates how to apply extrusion to a renderer on a feature layer.
+Extrude features based on their attributes.
 
-![Feature Layer Extrusion App](feature-layer-extrusion.png)
+![Image of feature layer extrusion](feature-layer-extrusion.png)
 
-## Features
+## Use case
 
+Extrusion is the process of stretching a flat, 2D shape vertically to create a 3D object in a scene. For example, you can extrude building polygons by a height value to create three-dimensional building shapes.
+
+## How to use the sample
+
+Press the button to switch between using population density and total population for extrusion. Higher extrusion directly corresponds to higher attribute values.
```

Comment:
let's keep this description the same as the java one:

> Tap the toggle to switch between using population density and total population for extrusion. Higher extrusion directly corresponds to higher attribute values.

Reply:
Don't know how I missed that!